File: JAVA/125. Valid Palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
        s=s.toLowerCase();
        String a="";
        int i;
        for(i=0;i<s.length();i++){
            if((int)s.charAt(i)>96 && (int)s.charAt(i)<123||(int)s.charAt(i)> 47 && (int)s.charAt(i) < 58)
            a+=s.charAt(i);
        }
        for(i=0;i<a.length()/2;i++){
            if(a.charAt(i)!=a.charAt(a.length()-i-1))
            return false;
        }
        return true;
    }
}

File: C++/125. Valid Palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        transform(s.begin(),s.end(),s.begin(),::tolower);
        string str="";
        int i;
        for(i=0;i<s.size();i++){
            if((int)s[i]>96 && (int)s[i]<123||(int)s[i] > 47 && (int)s[i] < 58)
            str+=s[i];
        }
        for(i=0;i<str.size()/2;i++){
            if(str[i]!=str[str.size()-i-1])
            return false;
        }
        return true;
    }
};